kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/config/tls/FilePassword.java

```diff
@@ -14,34 +14,42 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 
+import edu.umd.cs.findbugs.annotations.Nullable;
+
 /**
  * A reference to the file containing a plain text password in UTF-8 encoding.  If the password file
  * contains more than one line, only the characters of the first line are taken to be the password,
  * excluding the line ending.  Subsequent lines are ignored.
  *
- * @param filePath file containing the password,
+ * @param passwordFile file containing the password,
  */
-public record FilePassword(String filePath) implements PasswordProvider {
+public record FilePassword(String passwordFile) implements PasswordProvider {
```

Comment:
My hope was that we could annotate the formal parameter with a `@JsonAlias("filePath")` and that would handle the alternative property name.  Unfortunately, Jackson Databind doesn't seem to support that possibility.

https://github.com/FasterXML/jackson-databind/issues/4327

The least intrusive work around was to introduce another class into the type hierarchy, FilePasswordFilePath,  that understands the old name.  Yuck!

Reply:
That does work, but it doesn't play too well with the Sundrio Builders.  I want to keep filePath out of the built API.  
@ExternalBuilders does let you pass an `ignore` but that doesn't seem to work properly.   I think FilePasswordFilePath  is least worst.  Hopefully https://github.com/FasterXML/jackson-databind/issues/4327 will
get some traction and the class can be got rid of later.